cmd/invoke.go

```diff
@@ -79,7 +83,13 @@ dapr invoke --unix-domain-socket --app-id target --method sample --verb GET
 			}
 		}
 
-		response, err := client.Invoke(invokeAppID, invokeAppMethod, bytePayload, invokeVerb, invokeSocket)
+		var response string
+		if kubernetesMode {
+			response, err = kubernetes.Invoke(invokeAppID, invokeAppMethod, bytePayload, invokeVerb)
+		} else {
+			client := standalone.NewClient()
+			response, err = client.Invoke(invokeAppID, invokeAppMethod, bytePayload, invokeVerb, invokeSocket)
+		}
```

Comment:
nit: line break

Reply:
How can I fix this?
`make lint` without any hints.

Reply:
Oh you can simply add a new line after line 92. It's only for readability purpose.

Reply:
OK, I fixed it.